splink/redshift/redshift_helpers/redshift_transforms.py

```diff
@@ -0,0 +1,14 @@
+import sqlglot
+from sqlglot import expressions as exp
+
+
+def cast_concat_as_varchar(node):
```

Comment:
Unused function

Reply:
Yep - a hangover from copying across the postgres linker I used as a starting point